DataStructures/HashMap/Hashing/HashMap.java

```diff
@@ -13,39 +21,62 @@ public HashMap(int hsize) {
         this.hsize = hsize;
     }
 
-    public int hashing(int key) {
-        int hash = key % hsize;
-        if (hash < 0)
-            hash += hsize;
-        return hash;
-    }
-
```

Comment:
un-delete this

Reply:
Actually for this thing I've change it to private